docs/docs/configuration/mapper.mdx

```diff
@@ -185,3 +185,11 @@ dotnet_diagnostic.RMG020.severity = error # Unmapped source member
 ### Strict enum mappings
 
 To enforce strict enum mappings set `RMG037` and `RMG038` to error, see [strict enum mappings](./enum.mdx).
+
+## Default Mapper configuration
+
+The `DefaultMapperAttribute` allows to set default configurations applied to all mappers on the assembly level.
+
+```csharp
+[assembly: Riok.Mapperly.Abstractions.DefaultMapperAttribute(EnumMappingIgnoreCase = true)]
```

Comment:
IMO using the short name should be enough here, and makes it also simpler to read `[assembly: MapperDefaults(EnumMappingIgnoreCase = true)]`

Reply:
Fixed.